docs/Contributors_Guide/github_workflow.rst

```diff
@@ -3,246 +3,349 @@
+
+  replacing <your-github-user> with your GitHub username.
+
+* Change directories to the METplus directory:
+
+  .. code-block:: none
+
+    cd METplus
+
+  Now you are in your local METplus repository.
+
+Set upstream remote
+^^^^^^^^^^^^^^^^^^^
+
+* Add a remote named origin to the clone of your local Git repository, which
+  will allow you to push changes to the repository you forked in step 1.
+
+  .. code-block:: none
+		  
+    git remote add upstream https://github.com/dtcenter/METplus
+    
+* To verify that the upstream and origin are correct, at the command line enter:
+
+  .. code-block:: none
+
+    git remote -v
+
+  You should see something like the following:
+
+  .. code-block:: none
+
+    origin	https://github.com/<your-github-user>/METplus (fetch)
+    origin	https://github.com/<your-github-user>/METplus (push)
+    upstream	https://github.com/dtcenter/METplus (fetch)
+    upstream	https://github.com/dtcenter/METplus (push)
+
+  where <your-github-user> is your GitHub username.
+
+
+Generate a feature branch from the 'develop' branchfor new development
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Create a feature branch in the NCAR/METplus GitHub repository following this naming convention
+
+| *feature_<Github Issue number>_<brief_description>*
 |
+
+  For example, for GitHub issue #777 that creates new wrapper xyz, the feature branch would be named:
+
+| *feature_777_wrapper_xyz*
 |
-| *Re-cap*:
-|   You've created a feature branch, made changes, committed those changes to the repository, pushed them to GitHub,
-|   opened a pull request,had your changes merged by the repository maintainers, and finally performed some clean-up.
+
+* Create your feature branch based off the develop branch
+
+  .. code-block:: none
+
+    git checkout develop
+
+* Verify that you are currently working in the develop branch by running
+
+  .. code-block:: none
+
+    git branch
+
+  You should see something like the following:
+
+  .. code-block:: none
+
+    * develop
+    main_v3.1
+
+  The asterisk, "*", indicates the currently active branch.
+
+* At the command line, create and checkout the feature branch. For example:
+
+  .. code-block:: none
+		  
+    git checkout -b feature_777_wrapper_xyz
+
+  replacing *feature_777_wrapper_xyz* with your feature branch name.
+
+* Verify that you are working in the correct branch by running:
+
+  .. code-block:: none
+
+    git branch
+
+  You should see something like the following:
+
+  .. code-block:: none
+		  
+    develop
+    main_v3.1
+    * feature_777_wrapper_xyz
+
+  where the asterisk, "*", indicates which branch is currently in use/checked out.
+
+
+Make changes to code in the feature branch
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users should make changes to their feature branch and commit changes to their
+local repository.
+
+* Create code following the coding standards in the Coding Standards section of
```

Comment:
Consider creating a link to the Coding Standards section

Reply:
Thanks!  Done!  Still need to commit this change.

Reply:
I have committed it.